packages/react-components/react-message-bar/library/src/components/MessageBar/useMessageBarReflow.ts

```diff
@@ -27,7 +27,8 @@ export function useMessageBarReflow(enabled: boolean = false) {
       }
 
       const entry = entries[0];
-      const borderBoxSize = entry?.borderBoxSize[0];
+      // `borderBoxSize` is not supported before Chrome 84, Firefox 92, nor Safari 15.4
```

Comment:
For Fluent's [support matrix](https://react.fluentui.dev/?path=/docs/concepts-developer-browser-support-matrix--page) only Safari is relevant.

Reply:
As mentioned in chat, that matrix doesn't align with Outlook or M365's support matrix.